src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    @Override
+    public List<String> solveReport(String filePath) {
+        FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
Do not create Dao each time you call this method. Create only one Dao for this class